tls-utils/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "solana-tls-utils"
+description = "Solana TLS utilities"
+documentation = "https://docs.rs/solana-tls-utils"
+publish = false
+
```

Comment:
extra line here

Reply:
publish=false will have to go away after this is merged as far as I understand.

Reply:
ya are we going to publish this on crates.io? or no?

Reply:
I have no idea if anyone would ever use it there. Probably there is no benefit to that in this case.